scripts/process.py

```diff
@@ -1,61 +1,70 @@
+from dataflows import Flow, PackageWrapper, ResourceWrapper, validate
+from dataflows import add_metadata, dump_to_path, load, set_type, printer
 import os
 import urllib.request
 import logging
 import csv
 
 downloaded = 'cache/bbk_WU5500.csv'
-outpath = 'data/data.csv'
 source_url = 'http://www.bundesbank.de/cae/servlet/StatisticDownload?tsId=BBEX3.M.XAU.USD.EA.AC.C06&its_csvFormat=en&its_fileFormat=csv&mode=its'
 
-def download():
+
+def download_csv_resource():
```

Comment:
Why do we need to download at all? we can load the data with `load(url)` processor

Reply:
By going to the specified url, my browser starts the download automatically, I don't think `load(url)` can work for this case.

Reply:
@Branko-Dj have you tried?

Reply:
I tried loading a resource but it doesn't work as (I presume) there is some redirection/server process involved. Here is the link I'm using to download the data for gold priceshttp://www.bundesbank.de/cae/servlet/StatisticDownload?tsId=BBEX3.M.XAU.USD.EA.AC.C06&its_csvFormat=en&its_fileFormat=csv&mode=its. In contrast to this one here is the link we used for [bond-yield](https://github.com/datasets/bond-yields-uk-10y) repo http://www.bundesbank.de/cae/servlet/StatisticDownload?tsId=BBEX3.M.XAU.USD.EA.AC.C06&its_csvFormat=en&its_fileFormat=csv&mode=its.

Using load processor on for bond-yield works with no problems but it won't work for gold prices link

Reply:
@zelima ok thanks, I made a mistake in my analysis. It will load and read data but it won't dump it because there is a superfluous empty column at the end. There are a few issues preventing me from completing this which I will try to resolve now

Reply:
FIXED. Data is loaded without downloading it and empty column is deleted